crates/turborepo-vercel-api/src/lib.rs

```diff
@@ -124,3 +124,32 @@ pub struct APIError {
     pub code: String,
     pub message: String,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum CacheSource {
+    Local,
+    Remote,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum CacheEvent {
+    Hit,
+    Miss,
+}
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct AnalyticsEvent {
+    pub session_id: Option<String>,
+    pub source: CacheSource,
+    pub event: CacheEvent,
+    pub hash: String,
+    pub duration: u64,
+}
```

Comment:
Can we have a unit test to make sure this serializes how we expect it to serialize?